conf/vertebrates/mlss_conf.xml

```diff
@@ -34,98 +34,285 @@
 
   <pairwise_alignments>
 
-    <!-- First, our top four species, which are references for all chordates -->
+    <!-- Mammals -->
+    <!-- Use of "default" collection excludes the mouse strains and the pig breeds -->
+    <!-- NB: we have a CACTUS alignment for mouse strains and an EPO Extended MSA for pig breeds -->
     <one_vs_all method="LASTZ_NET" ref_genome="homo_sapiens">
-      <!-- "default" collection means that the mouse strains are excluded (because only the reference mouse deserves to be aligned to human) -->
       <species_set in_collection="default">
-        <taxonomic_group taxon_name="Chordata"/>
+        <taxonomic_group taxon_name="Mammalia"/>
+        <!-- Remove all the taxa that have a better reference than human -->
+        <taxonomic_group taxon_name="Rodentia" exclude="1"/>
+        <taxonomic_group taxon_name="Carnivora" exclude="1"/>
+        <taxonomic_group taxon_name="Perissodactyla" exclude="1"/>
+        <taxonomic_group taxon_name="Artiodactyla" exclude="1"/>
+        <taxonomic_group taxon_name="Metatheria" exclude="1"/>
+        <taxonomic_group taxon_name="Prototheria" exclude="1"/>
       </species_set>
     </one_vs_all>
-    <one_vs_all method="LASTZ_NET" ref_genome="mus_musculus" ref_amongst="Chordata">
-      <!-- "default" collection means that the mouse strains are excluded (because we have the Cactus alignment for those) -->
+
+    <one_vs_all method="LASTZ_NET" ref_genome="mus_musculus">
       <species_set in_collection="default">
         <taxonomic_group taxon_name="Rodentia"/>
       </species_set>
     </one_vs_all>
-    <one_vs_all method="LASTZ_NET" ref_genome="gallus_gallus" against="Sauropsida" ref_amongst="Chordata"/>
-    <one_vs_all method="LASTZ_NET" ref_genome="danio_rerio" ref_amongst="Chordata">
-      <species_set>
-        <!-- Include urochordates, cephalochordates, cyclostomes, cartilaginous fishes, ray-finned fishes, coelacanths and lungfishes -->
-        <taxonomic_group taxon_name="Chordata"/>
-        <taxonomic_group taxon_name="Tetrapoda" exclude="1"/>
+
+    <one_vs_all method="LASTZ_NET" ref_genome="canis_lupus_familiaris" against="Carnivora"/>
```

Comment:
Following https://github.com/Ensembl/ensembl-compara/pull/199#issuecomment-644611467 let's keep the e101 rules and edit them if cat / dog changes
```suggestion
    <one_vs_all method="LASTZ_NET" ref_genome="canis_lupus_familiaris" against="Caniformia"/>
    <one_vs_all method="LASTZ_NET" ref_genome="felis_catus" against="Feliformia"/>
```

Reply:
I had a call with @JAlvarezJarreta to talk about this one. The principle is to move to the new rules that don't necessitate any new LastZ, and keep the other ones for now. We already have have dog vs all carnivores, so we can remove cat vs all feliformes

Reply:
My understanding is that if the change of the rules (i.e. reduce the number of references) doesn't generate new LastZs, then they should be applied now.